pkg/kubectl/cmd/create_service_test.go

```diff
@@ -48,7 +48,7 @@ func TestCreateService(t *testing.T) {
 	buf := bytes.NewBuffer([]byte{})
 	cmd := NewCmdCreateServiceClusterIP(f, buf)
 	cmd.Flags().Set("output", "name")
-	cmd.Flags().Set("tcp", "8080:8000")
+	cmd.Flags().Set("ports", "tcp:8080:8000")
```

Comment:
refactor the test case to table-driven tests for testing multiple scenarios (tcp/udp/invalid) ?